tasks/new_e2e_tests.py

```diff
@@ -91,7 +91,7 @@ def run(
         test_run_arg = f"-run {test_run_name}"
 
     cmd = f'gotestsum --format {gotestsum_format} '
-    cmd += '{junit_file_flag} --packages="{packages}" -- -ldflags="-X {REPO_PATH}/test/new-e2e/tests/containers.GitCommit={commit}" {verbose} -mod={go_mod} -vet=off -timeout {timeout} -tags {go_build_tags} {nocache} {run} {skip} {coverage_opt} {test_run_arg} -args {osversion} {platform} {arch} {cws_supported_osversion} {keep_stacks}'
```

Comment:
❓ 
- Why are we completely removing the build tags from the command? Isn't it possible that we use them at some point?
- If we are removing the build tags completely from the command, shouldn't we also remove `'tags': 'Build tags to use',` from the help message of the invoke function, and also from the function arguments? Otherwise it might be confusing for the user.

Reply:
My bad : I didn’t realize that `go_build_tags` was supposed to be set by the `tags` functions parameter.